internal/core/data/event.go

```diff
@@ -88,7 +89,7 @@ func getEvents(limit int) ([]contract.Event, error) {
 	return events, err
 }
 
-func addNewEvent(e models.Event, ctx context.Context, loggingClient logger.LoggingClient) (string, error) {
+func addNewEvent(e models.Event, ctx context.Context, loggingClient logger.LoggingClient, dbClient interfaces.DBClient) (string, error) {
```

Comment:
Nitpick - lines > ~120 characters should be folded; i.e. 

``` go
func addNewEvent(
	e models.Event, 
	ctx context.Context,
	loggingClient logger.LoggingClient,
	dbClient interfaces.DBClient) (string, error) {

	err := checkDevice(e.Device, ctx)
	if err != nil {
		return "", err
	}
        ...
```

Comment once; occurs elsewhere.

Reply:
I am going to create an issue to add this to an `editorconfig` so that this is consistent rather than doing this by hand.